src/frontend/src/views/LandingPageV2/components/Footer/index.tsx

```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
 import { useEffect } from "react";
-import BiohubLogoImg from "src/common/images/cz-biohub-logo.png";
```

Comment:
It seems like you replaced a png with an svg, which actually does not work as a direct swap out. With this code, the logo is not visible.
![Screenshot 2023-03-28 at 9 46 37 AM](https://user-images.githubusercontent.com/7562933/228311132-1c8386b0-7359-4495-a4e2-ccc2d1666a7d.png)

Reply:
Replaced with png version 😄  thanks for the heads up!